src/compiler/base/nodes/tags/step.test.ts

```diff
@@ -91,19 +107,29 @@ describe("step tags", async () => {
           Correct the statement taking into account the reason: '{{ analysis.reason }}'.
         {{ endif }}
       </step>
-    `);
+    `)
 
-    const chain = new Chain({ prompt, parameters: { mock }});
-    const error = await getExpectedError(() => complete({ chain, callback: async () => `
+    const chain = new Chain({ prompt, parameters: { mock } })
+    let error: CompileError
+    try {
+      await complete({
+        chain,
+        callback: async () =>
+          `
       Bad JSON.
-    `.trim()}), CompileError)
-    expect(error.code).toBe('invalid-step-response-format')
+    `.trim(),
+      })
+    } catch (e) {
+      error = e as CompileError
+      expect(e).toBeInstanceOf(CompileError)
```

Comment:
This is the change. The abstraction we have `getExpectedError` was not working in this case so I just made the plain try/catch here